server/routerlicious/packages/services-utils/src/tokenManager.ts

```diff
@@ -0,0 +1,147 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { NetworkError } from "@fluidframework/server-services-client";
+import { IWebSocket } from "@fluidframework/server-services-core";
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+
+/**
+ * Interface of web socket tracker
+ * it tracks the mapping of web socket and token used to establish the socket connection
+ */
+export interface IWebSocketTracker {
```

Comment:
I suggest adding these types to services-core and the functions/classes to services-client. services-utils is not isomorphic and _cannot be used_ in lambdas

Reply:
Thank you for the suggestion. Will make the change.